src/main/java/seedu/address/logic/util/ModuleClassModificationUtil.java

```diff
@@ -0,0 +1,119 @@
+package seedu.address.logic.util;
+
+import static seedu.address.logic.commands.AddLessonCommand.MESSAGE_EXISTING_LESSON;
+import static seedu.address.logic.commands.EditLessonCommand.MESSAGE_DUPLICATE_LESSON;
+
+import java.util.ArrayList;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+import java.util.UUID;
+
+import seedu.address.commons.core.index.Index;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.components.name.Name;
+import seedu.address.model.lesson.Lesson;
+import seedu.address.model.moduleclass.ModuleClass;
+
+/**
+ * Contains utility methods for modifying {@code Lessons} in {@code ModuleClass}.
+ */
+public class ModuleClassModificationUtil {
+
+    /**
+     * Adds all lessons in the target module class to the new module class.
+     * The {@code lessonToAdd} is added to the new module class.
```

Comment:
```suggestion
     * Adds {@code lessonToAdd} to {@code moduleClassToAddTo}.
     * All existing {@code Lesson}s in {@code moduleClassToAddTo} are copied to the new {@code ModuleClass}.
```

Perhaps we should also change `moduleClassToAddTo` to `targetModuleClass` for consistency?